Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,56 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-558-ga70c9ba-src.tar.gz"
+  sha256 "a07cb6cc92bb7f05cc35f738bbde627b655df62ac7dbcd272716581c7b6bf5b1"
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
+    version "1.6.8"
+  end
+
+  head do
+    url "https://github.com/tarantool/tarantool.git", :branch => "master", :shallow => false
+  end
+
+  option "with-debug", "Build Debug version"
+
+  depends_on "cmake"    => :build
+  depends_on "readline" => :build
+
+  def install
+    args = []
+
+    if build.with? "debug"
+      ENV.enable_warnings
+      ENV.deparallelize
+      args << "-DCMAKE_BUILD_TYPE=Debug"
+      ohai "Building with Debug"
+    else
+      args << "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+      ohai "Building with Release"
+    end
+
+    args << "-DCMAKE_INSTALL_SYSCONFDIR=#{prefix}/etc"
+    args << "-DCMAKE_INSTALL_LOCALSTATEDIR=#{prefix}/var"
+    args << "-DCMAKE_INSTALL_MANDIR=#{doc}"
+    args += std_cmake_args
+
+    system "cmake", ".", *args
+    system "make"
+    system "make", "install"
+    doc.install "extra/dist/example.lua"
+  end
+
+  def caveats; <<-EOS.undent
+    Add user "tarantool" or provide alternative user in your configuration script for Tarantool.
```

Comment:
Instead make `post_install` change the configuration so that it runs as `ENV["$USER"]`